pinot-clients/pinot-java-client/src/main/java/org/apache/pinot/client/BrokerSelector.java

```diff
@@ -25,4 +25,8 @@
    * @return
    */
   String selectBroker(String table);
+
+  default void close() {
```

Comment:
Suggest making it not default to enforce all the implementation to close the resources (it could be empty if it does not hold any resource)